docs/getting-started/codacy-quickstart.md

```diff
@@ -4,9 +4,16 @@ description: Set up Codacy to automatically analyze your source code and identif
 
 # Codacy quickstart (5 min)
 
-Codacy automatically analyzes your source code and identifies issues as you go, helping you develop software more efficiently with fewer issues down the line. Through static code review analysis, Codacy notifies you of security issues, code coverage, code duplication, and code complexity in every commit and pull request.
+Codacy is an automated code quality and coverage platform that analyzes your source code and identifies issues as you go, helping your team ship robust software by scanning over 40 programming languages, such as JavaScript, Python, Java, C#, and PHP.
 
-Check out our product demo for an overview of the main Codacy features (recorded on February 4, 2022):
+By integrating with your Git provider, Codacy keeps track of your team’s work, analyzes relevant commits, highlights problems, suggests improvements, and protects your codebase from unwelcome changes. Codacy monitors the following metrics across your projects, from organization and repository overviews to individual files, pull requests, and commits:
```

Comment:
IMO, it would be clearer to have "monitors the following metrics across your projects" just before the items list.

For example (not sure if "level" is the best word):

```suggestion
By integrating with your Git provider, Codacy keeps track of your team’s work, analyzes relevant commits, highlights problems, suggests improvements, and protects your codebase from unwelcome changes. From organization and repository level to individual files, pull requests, and commits, Codacy monitors the following metrics across your projects:
```